resources/public/api/conf/2.0/testdata/fund-release-outcome.md

```diff
@@ -279,7 +281,8 @@
         <pre class="code--block">
 {
   "code": "FUND_RELEASE_SUPERSEDED",
-  "message": "This fund release has already been superseded"
+  "message": "This fund release has already been superseded",
+  "lifeEventId": "5678900001"
```

Comment:
As with previous comment, I think this should be the ID for the fund release, please can you confirm with Trevor?

Reply:
I have made the changes, can you please review. Thanks :)